src/pages/people/tabs/Wanted.tsx

```diff
@@ -107,12 +107,8 @@ export const Wanted = observer(() => {
               }}
               widget={'wanted'}
               onSucces={() => {
-                history.goBack();
```

Comment:
can you post a video on the `onSucces` seems like we only saw the `onGoBack` cause I think the `onSucces` takes us to the bounties page and we might want to go back in this scenario